### PR TITLE
Third dot + tooltips and links

### PR DESCRIPTION
**Pull request by slavingia**
- [x] Updated nav to remove auth, moved to 1st dot
- [x] Second dot points to Discord
- [x] Added tooltip texts to foreshadow functionality
- [x] 3rd dot points to Polygonscan 
- [x] Light up 3rd dot by reading from Polygonscan
- [x] Message if not connected to Polygon

### CODE REVIEW

**Comment by slavingia** [comment]
#### 📝 Changed routes:


| [**/**](https://fweb3-1tf234gva-shlvc.vercel.app/) |
|:-:|
|<a href="https://fweb3-1tf234gva-shlvc.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/-QBmHYWv3yJp_uEAXbO_ngw9FTHCheLkO03vjEhGF6B9JzKunj4YAaFPux4Pdm78679o1BrNUeSwwiyGn7wYVA.png">(view full size)</a> |







Commit 9f072b9ea0c88aeff2b50ad15d76425682e9c8ee (https://fweb3-1tf234gva-shlvc.vercel.app).

File: pages/api/polygon.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+
+export default async function handler(req, res) {
+  const response = await fetch("https://api.polygonscan.com/api?module=account&action=txlist&address=" + req.query.wallet_address + "&startblock=0&endblock=99999999&page=1&offset=10&sort=asc&apikey=" + process.env.POLYGON_API_KEY);
+  const json = await response.json();
+
+  let hasUsedFaucet = false;
+
+  for (let i = 0; i < json.result.length; i++) {
+    if (json.result[i]["to"] == "0x67806adca0fd8825da9cddc69b9ba8837a64874b") {
```
Comment (slavingia) [inline pages/api/polygon.js:10]: Do we want to be more explicit about what we're looking for here? Currently there's only one function to call so I think it's fine, but maybe worth double-checking.

Comment (Bschuster3434) [reply pages/api/polygon.js:10]: We need to make sure that the faucet call was successful and didn't error.

Example of a call that failed: https://polygonscan.com/tx/0x204e155350c3f06ebcce9c69c7edee0ab1ca5a99dcf30fe486bb9fe5327d1030

If it fails, do we catch that?

Comment (slavingia) [reply pages/api/polygon.js]: Updated to check for failures.

**Pull request merged**